deliver/lib/deliver/runner.rb

```diff
@@ -86,10 +88,15 @@ def verify_version
 
     # Upload all metadata, screenshots, pricing information, etc. to iTunes Connect
     def upload_metadata
+      upload_metadata = UploadMetadata.new
+      upload_screenshots = UploadScreenshots.new
+
       # First, collect all the things for the HTML Report
-      screenshots = UploadScreenshots.new.collect_screenshots(options)
-      UploadMetadata.new.load_from_filesystem(options)
-      UploadMetadata.new.assign_defaults(options)
+      screenshots = upload_screenshots.collect_screenshots(options)
+      upload_metadata.load_from_filesystem(options)
```

Comment:
@ohayon This MAY need to get moved to a different PR or talked about further. Moving this out makes a lot of the tests fail because the tests are written and passed different things via the method parameters so would require reorganizing tests which seems a little much for this PR 😇  I'm totes down for doing this in another PR though 💪